package.json

```diff
@@ -24,7 +24,10 @@
     "node": ">=0.10"
   },
   "scripts": {
-    "install": "prebuild --install",
+    "build:libzmq": "node scripts/download-win-lib.js && ./build_libzmq.sh",
```

Comment:
You'll have to write a node script. If Windows batch commands allowed inline comments, there would be a hacky way (I tried using the `::` empty label trick). It would be much better to maintain js though.